extensions/BugModal/web/bug_modal.css

```diff
@@ -1067,9 +1075,10 @@ h3.change-name a {
 }
 
 #floating-message {
-  position: absolute;
+  position: fixed;
   left: 50%;
-  margin: 4px;
+  top: 50px;
+  z-index: 100;
```

Comment:
Previously the floating message could be scrolled off the top of the screen; a fixed position fixes that.